### PR TITLE
Step 2: Public Feed

### PR DESCRIPTION
**Pull request by elrick97**
Hi guys, just creating this pull request to start familiarizing with this. This new portion of code includes the second step on the public feed. :) Have a great day!

### CODE REVIEW

File: src/main/webapp/WEB-INF/appengine-web.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 <appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
   <application>sp19-codeu-5-328</application>
-  <version>1</version>
+  <version>rick</version>
```
Comment (tbiscuit) [inline src/main/webapp/WEB-INF/appengine-web.xml]: Should put this back to 1 before pulling. 
```suggestion
  <version>1</version>
```

File: target/codeu-starter-project-0.0.1-SNAPSHOT/WEB-INF/appengine-web.xml [hunk 2]
```diff
@@ -16,8 +16,8 @@
 -->
 
 <appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
-  <application>YOUR_PROJECT_ID</application>
-  <version>1</version>
+  <application>sp19-codeu-5-328</application>
+  <version>rick</version>
```
Comment (tbiscuit) [inline target/codeu-starter-project-0.0.1-SNAPSHOT/WEB-INF/appengine-web.xml]:
```suggestion
  <version>1</version>
```

**Review by tbiscuit — CHANGES_REQUESTED** [review]
Nice! Couple minor requests. :)

**Comment by elrick97** [comment]
Did the changes in my last commit. :)

**Pull request closed without merging**